projects/plugins/jetpack/tests/e2e/specs/free-blocks.test.js

```diff
@@ -1,11 +1,11 @@
-import BlockEditorPage from '../lib/pages/wp-admin/block-editor';
-import PostFrontendPage from '../lib/pages/postFrontend';
-import PinterestBlock from '../lib/pages/wp-admin/blocks/pinterest';
-import EventbriteBlock from '../lib/pages/wp-admin/blocks/eventbrite';
+import BlockEditorPage from 'jetpack-e2e-commons/pages/wp-admin/block-editor';
```

Comment:
Do you think it would make sense to provide named exports for pages, env, reports, etc? So instead importing pages one by one, we could do something like: 

```
import { BlockEditorPage, PostFrontendPage, PinterestBlock } from 'jetpack-e2e-commons/pages';
```

Reply:
I love the idea and I spent some time on it, as it should be pretty straightforward. Turns out it's not and it got really messy really quick. I'd rather leave this in a follow-up PR.